x-pack/plugins/security_solution/public/common/components/event_details/table/action_cell.tsx

```diff
@@ -51,22 +52,9 @@ export const ActionCell: React.FC<Props> = React.memo(
       values,
     });
 
+    const { closeTopN, toggleTopN, showTopN } = useTopNPopOver(setIsPopoverVisible);
     const { aggregatable, type } = fieldFromBrowserField || { aggregatable: false, type: '' };
-
-    const [showTopN, setShowTopN] = useState<boolean>(false);
     const { timelineId: timelineIdFind } = useContext(TimelineContext);
-    const [hoverActionsOwnFocus] = useState<boolean>(false);
```

Comment:
that was weird?

Reply:
Yeah, I don't know why we should keep that value as a state without the ability to change it... I've tried to check the intense in the original PR which added this, but still cannot explain. @michaelolo24 do you have any thoughts about this?

Reply:
Hey, this is weird. I may have added it with the intention that it could be updated, but didn't revert it when I realized it wouldn't be. Thanks for removing!